Add instructions to gen section (#66)
GatorEducator/gatorgrade: README.md

### Generating a gatorgrade.yml file

The generation of yml file can be used to create assignments.

In the terminal within the main directory.

Use command `gatorgrade generate <path> <root directory>` to gen a yml file.

- User should provide precise names of folders or files.

- The user should also provide correct character case as input.

> - All folder(s) containing files starting with `__` or `.` will be ignore

Reviewer: ```suggestion
- All folder(s) containing files starting with `__` or `.` will be ignored
```

- Don't name folder or file starting with dot or double underscore.

> - These files will be ignored.

Reviewer: This is a duplicate of the point above.

> All sub-dirs and files will be reserved except ones starting with `.` or `__`

Reviewer: It's not clear what "reserved" means here - could you clarify it?

Author: Thanks!

### Generating a gatorgrade.yml file

The generation of yml file can be used to configure tests.

> 

Reviewer 2: Could you please try to simplify the messages I show below? They are what users need to know for calling the implementation function.
     Warning: User should provide precise names of folders or files and provide correct character case as input
     Warning: The empty folder or the folders only containing files starting with __ or . will be ignore
     Warning: Don't name folder or file starting with dot or double underscore. They will be ignored
     If you input a folder name, all the sub-dirs and files will be reserved except the ones starting with . or __
     Warning: Only folder names in root dir or the one after root will be checked. Don't put the target folders too deep
     The file names have no above problem. You can put it in any dir.

Reviewer 2: Also, rn here is another input argument. User can provide relative directory to decide where get started to scan

### Generating a gatorgrade.yml file

The generation of yml file can be used to create assignments.

In the terminal within the main directory.

> Use command gatorgrade generate <TARGET_PATH_LIST> to generate a yml file.

Reviewer: You'll probably still want the backticks ``` around this command.

> - User should provide precise name of the of target list .

Reviewer: ```suggestion
- User should provide full paths in the target list .
```

- The user should also provide correct character case as input.

- All folder(s) containing files starting with `__` or `.` will be ignored

- Don't name folder or file starting with dot or double underscore.

> All sub-dirs and files will be stored except ones starting with `.` or `__`

Reviewer: These three points are redundant.
